server/src/main/java/io/spine/server/GrpcContainer.java

```diff
@@ -117,15 +120,41 @@ private GrpcContainer(Builder builder) {
     /**
      * Starts the service.
      *
+     * <p>A gRPC-default executor is used for the server routines. I.e., in-process gRPC server
```

Comment:
Please reformat into a list:
```
A gRPC-default executor is used for the server routines:
 1. In-process gRPC server is going to ...
 2. A server exposed at a port will ...
```

It reads clearer and avoids the awkward "I.e." at the beginning of a sentence.